plugins/virt.yaml

```
@@ -35,11 +35,11 @@ spec:
         - from: "/virtctl/virtctl-v0.17.2-darwin-amd64"
           to: "virtctl"
       bin: "virtctl"
-  shortDescription: controls virtual machine related operations.
+  shortDescription: Control KubeVirt Virtual Machines
```

I understand the camel-case for "KubeVirt", but "virtual machines" is just plain English and can be lower-case, right? Also, I think the short description could hint that this is a wrapper around `virtctl`
```suggestion
  shortDescription: Control KubeVirt virtual machines using virtctl
```

---

Both good points, I'll address them. Thanks.

---

(Added a new commit as it suites my flow better)